src/core/public/chrome/ui/header/header.tsx

```diff
@@ -109,55 +113,46 @@ export function Header({
-            sections={[
-              {
-                items: [
-                  <HeaderLogo
-                    href={homeHref}
-                    forceNavigation$={observables.forceAppSwitcherNavigation$}
-                    navLinks$={observables.navLinks$}
-                    navigateToApp={application.navigateToApp}
-                    branding={branding}
-                  />,
-                  <LoadingIndicator loadingCount$={observables.loadingCount$} />,
-                ],
-                borders: 'none',
-              },
-              {
-                ...(observables.navControlsCenter$ && {
-                  items: [
-                    <EuiShowFor sizes={['m', 'l', 'xl']}>
-                      <HeaderNavControls navControls$={observables.navControlsCenter$} />
-                    </EuiShowFor>,
-                  ],
-                }),
-                borders: 'none',
-              },
-              {
-                items: [
-                  <EuiHideFor sizes={['m', 'l', 'xl']}>
-                    <HeaderNavControls navControls$={observables.navControlsCenter$} />
-                  </EuiHideFor>,
-                  <HeaderHelpMenu
-                    helpExtension$={observables.helpExtension$}
-                    helpSupportUrl$={observables.helpSupportUrl$}
-                    opensearchDashboardsDocLink={opensearchDashboardsDocLink}
-                    opensearchDashboardsVersion={opensearchDashboardsVersion}
-                  />,
-                  <HeaderNavControls navControls$={observables.navControlsRight$} />,
-                ],
-                borders: 'none',
-              },
-            ]}
-          />
+          {useExpandedMenu && (
+            <EuiHeader
+              theme="dark"
+              position="fixed"
+              sections={[
+                {
+                  items: [<HeaderLogo {...branding} />],
+                  borders: 'none',
+                },
+                {
+                  ...(observables.navControlsExpandedCenter$ && {
+                    items: [
+                      <EuiShowFor sizes={['m', 'l', 'xl']}>
+                        <HeaderNavControls navControls$={observables.navControlsExpandedCenter$} />
+                      </EuiShowFor>,
+                    ],
+                  }),
+                  borders: 'none',
+                },
+                {
+                  ...((observables.navControlsExpandedCenter$ ||
+                    observables.navControlsExpandedRight$) && {
+                    items: [
+                      <EuiHideFor sizes={['m', 'l', 'xl']}>
+                        <HeaderNavControls navControls$={observables.navControlsExpandedCenter$} />
+                      </EuiHideFor>,
+                      <HeaderNavControls navControls$={observables.navControlsExpandedRight$} />,
+                    ],
+                  }),
+                  borders: 'none',
+                },
```

Comment:
This will be a breaking change for Bitergia - they'll need to update their menu plugin to uses these new navControls locations. But because we're going to the simplified menu by default, I think that's the best migration path (and better than trying to find some conditional solution) - any other plugins using the existing navControls locations won't need to do anything.